src/arrayMethodJoin.js

```diff
@@ -4,8 +4,21 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
+  [].__proto__.join2 = function(separator = ',') {
     // write code here
+    let finallString = '';
+
+    for (const item of this) {
```

Comment:
Maybe better use a simple for loop if you need index?

Reply:
Ok, i will change loop later, thanks))